Applications/SystemMonitor/main.cpp

```diff
@@ -83,7 +84,7 @@ static NonnullRefPtr<GUI::Widget> build_graphs_tab();
 class UnavailableProcessWidget final : public GUI::Frame {
     C_OBJECT(UnavailableProcessWidget)
 public:
-    virtual ~UnavailableProcessWidget() override {}
+    virtual ~UnavailableProcessWidget() override { }
```

Comment:
These changes are unrelated to the new `readlink` program and should be squashed into the previous commit.